packages/replay/src/types.ts

```diff
@@ -48,9 +48,11 @@ export interface WorkerResponse {
   id: number;
   method: string;
   success: boolean;
-  response: ReplayRecordingData;
+  response: unknown;
 }
 
+export type WorkerAddEventResponse = boolean;
```

Comment:
Ehh wouldn't mind a suggestion for a better name for this type alias. Right now it's just boolean for true/false if worker succeeds in processing the event, but could change in the future (e.g. we could return the current compressed size).

Reply:
IMHO this should return an object, otherwise it is hard to extend this in the future.
But actually, I wonder if we really need this at all. If all we care about is if it has been successfully added or not, it would be more appropriate to just let it reject the promise than to resolve with true.